broker/httpd/status.go

```diff
@@ -0,0 +1,38 @@
+package httpd
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/Symantec/Dominator/lib/html"
+)
+
+func (s *Server) statusHandler(w http.ResponseWriter, req *http.Request) {
```

Comment:
It way past time to start using the golang html/template.

Reply:
Codewise LGTM but we should talk in the office before a merge (we can defere templating and we could use JSON instead of YAML, but for configs YAML is soo much better)

Reply:
OK, let's talk shortly.